style.css

```diff
@@ -0,0 +1,63 @@
+body {
+  font-family: Roboto;
+}
+
+.calculator{
+  width: 248px;
+  height: 380px;
+  line-height: 1.5;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  transform: translate(-50%, -50%);
+  margin: auto;
```

Comment:
`margin: auto` is a different horizontal centering approach. Are you sure you need both? Are you sure both are doing what you expect? Try and remove any style declarations that are not explicitly needed.